src/makielayout/MakieLayout.jl

```diff
@@ -20,11 +20,15 @@ using GridLayoutBase
 using GridLayoutBase: GridSubposition
 import Showoff
 using Colors
+using Memoization: @memoize
+using LRUCache: LRU
 
 const FPS = Observable(30)
 const COLOR_ACCENT = Ref(RGBf(((79, 122, 214) ./ 255)...))
 const COLOR_ACCENT_DIMMED = Ref(RGBf(((174, 192, 230) ./ 255)...))
 
+const TICK_CACHE_SIZE = Ref{Int}(100)
```

Comment:
It just occurred to me that maybe this can't be dynamic given the `@memoize` is evaluated at precompile time?

Reply:
Yeah, it should be evaluated when the macro gets expanded..Could we make the LRU a const global, which can then be resized?